### ISSUE TITLE
ImgProxy 3.* returns 500 status for not found GCS images

### ISSUE THREAD

**Issue by scarfacedeb** [imgproxy/imgproxy#926]
## Description

ImgProxy 2.17.0 used to return the correct 404 status when it couldn't find any object in GCS.

ImgProxy 3.0.0 started to give back 500 for such cases instead:

```
Can't download source image: Status: 500; Can't download source image: Get "gs://bucket/obj.jpg": storage: object doesn't exist
```

There was [a similar PR](https://github.com/imgproxy/imgproxy/issues/822) for S3, but I don't think it's related.

Is there anything I can do to help with diagnostic or fixes?

**Comment by DarthSim** [comment]
Hey @scarfacedeb!
This issue is fixed in v3.7.0

**Comment by scarfacedeb** [comment]
Hey @DarthSim, thank you so much for the quick fix. 👍
